commands/deal.go

```diff
@@ -1,34 +1,136 @@
 package commands
 
 import (
+	"context"
+	"fmt"
+	"github.com/c2h5oh/datasize"
+	"github.com/filecoin-project/dealbot/lotus"
+	"github.com/filecoin-project/go-address"
 	"github.com/filecoin-project/go-fil-markets/storagemarket"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/big"
 	"github.com/filecoin-project/lotus/api"
+	"github.com/filecoin-project/lotus/chain/types"
 	"github.com/urfave/cli/v2"
 	"golang.org/x/xerrors"
+	"os"
+	"path"
 )
 
-var MakeStorageDeals = &cli.Command{
-	Name:  "MakeDeals",
+var MakeStorageDeal = &cli.Command{
+	Name:  "MakeDeal",
 	Usage: "Make storage deals with provided miners.",
 	Flags: []cli.Flag{
-		// TODO: parameters?
+		&cli.PathFlag{
+			Name:     "data-dir",
+			Usage:    "writable directory used to transfer data to node",
+			Aliases:  []string{"d"},
+			EnvVars:  []string{"DEALBOT_DATA_DIRECTORY"},
+			Required: true,
```

Comment:
I think we want 2 parameters for this:
where that directory is from the perspective of the dealbot, and where it is from the perspective of the lotus node.